### ISSUE TITLE
UI to inspect individual households and their members

### ISSUE THREAD

**Issue by aeickhoff** [citybound/citybound#151]
- [x] Show members
- [x] Show current member activity
   - [ ] ~~Transport time/current route?~~ de-scoped
- [x] Show household items
- [x] Show household & individual needs and resources
- [ ] ~~Show long-term planning (resource fulfillment/depletion rates)~~ de-scoped

**Comment by mallenwho** [comment]
Very basic peep (RCT term for Person or citizen, or sim) UI, put together using default word fonts and tools. Can reuse or develop your own. I think this looks ok. Covers the important parts of a person's character. Their demographics, links to their address or family. Their activity, and what they intend to do next (for a little bit of forward planning). Their emotions (important to know as mayor-god) and their dreams (because if they want to see an opera but you have no opera house, that is a player goal you can now do!)

Perhaps add to this a second tab to see all their emotions and feelings, in a Prison Architect-y way. But that would be a LARGE GUI, so maybe cherry pick the most significant ones. Job security, free time, health, money, and a choice 4 or 5 others.

I'd do something very similar for properties. Probably have two tabs. One for info on household, their budget (in simple terms), and how close they are to "leveling up" or whatever (crime rates, land values, traffic, other pull and push factors). In tab two put the house's inventory and "stats"

![sally hemmings](https://cloud.githubusercontent.com/assets/25683627/22931415/a3c9c68c-f318-11e6-8a15-f8dabec343aa.jpg)

**Comment by aeickhoff** [comment]
Depends on #171

**Comment by aeickhoff** [comment]
Some basics (household & member state and resources) were implemented in  b91b63b
